File: VENTUS_GeomKernel/Module_Assembly/src/GraphicInfo.cpp
#include "GraphicInfo.h" 

namespace ventus
{
  GraphicInfo::GraphicInfo() 
  { 
  }

  GraphicInfo::GraphicInfo(double R, double G, double B, double A)
      : mColor(R, G, B, A)
  {
  }

  GraphicInfo::~GraphicInfo() { }

} // ventus

File: UnitySTEPImporter/Assets/UnitySTEPImporter/Runtime/Scripts/ModuleAssembly/Mesh/GraphicInfo.cs
namespace VENTUS.StepImporter.ModuleAssembly.Mesh
{
    public class GraphicInfo
    {
		public GraphicInfo() { }

		public GraphicInfo(Color color)
		{
			this.color = color;
		}

		private Color color;
		public Color Color { get { return color; } set { color = value; } }
	}
}
